geeknote/gnsync.py

```diff
@@ -304,7 +305,20 @@ def _get_notes(self):
         Get notes from evernote.
         """
         keywords = 'notebook:"{0}"'.format(tools.strip(self.notebook_name))
-        return GeekNote().findNotes(keywords, 10000).notes
+        #print GeekNote().findNotes(keywords, 10000)
+        #try to get all notes in a notebook
+        #assum that the notebook has less than 10000 notes
```

Comment:
Why 10000? 
[dev.evernote] seems does not restrict on 10000?
`i32` is much larger.

[dev.evernote]: https://dev.evernote.com/doc/reference/NoteStore.html#Fn_NoteStore_findNotesMetadata